cmd/kubeadm/app/util/config/joinconfiguration.go

```diff
@@ -103,7 +103,9 @@ func documentMapToJoinConfiguration(gvkmap map[schema.GroupVersionKind][]byte, a
 		}
 
 		// verify the validity of the YAML
-		strict.VerifyUnmarshalStrict(bytes, gvk)
+		if err := strict.VerifyUnmarshalStrict(bytes, gvk); err != nil {
```

Comment:
@tedyu 
are you seeing this as a report from a certain static check too?

the same call is made here so the changes should be in sync:
https://github.com/kubernetes/kubernetes/blob/2036ee5ddeac235c0c39c705ad22cfa19bad309e/cmd/kubeadm/app/util/config/initconfiguration.go#L237

but the main thing we care about here is this klog warning:
https://github.com/kubernetes/kubernetes/blob/a856c7ab1de1bed266cbeab5dec703105d1bf2d0/cmd/kubeadm/app/util/config/strict/strict.go#L54

so i think we should ignore the error/output of this call.

/assign @rosti

Reply:
I was going over this class and noticed the discrepancy between the comment and no checking of the return value.

I saw the warning log.

But bytes is used by subsequent handling in documentMapToJoinConfiguration. If verification doesn't pass, I think there would be problem down the road.